CONTRIBUTING.md

```diff
@@ -92,6 +92,7 @@ npm run fix # Runs both of the above commands in parallel
 npm run test:watch # Runs karma on Chrome, re-running when source files change
 
 npm test # Lints all files, runs karma, and then runs coverage enforcement checks.
+npm run test:closure # Runs closure build tests against all closurized files
```

Comment:
Should closure be part of `npm run test`?

I suggest to make it so if not because I feel if TravisCI will be failed by closure test not pass, it should definitely be part of the test procedure that our contributors used to run.